tests/bud.bats

```diff
@@ -262,3 +262,25 @@ load helpers
   [ "$status" -eq 0 ]
   [ "$output" = "" ]
 }
+
+@test "bud-shell" {
+  target=alpine-image
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/shell
+  run buildah --debug=false inspect --type=image --format '{{.Docker.Config.Shell}}' ${target}
+  echo $output
+  [ "$status" -eq 0 ]
+  [ "$output" = "[/bin/sh -c]" ]
+  ctr=$(buildah from ${target})
+  run buildah --debug=false config --shell "/bin/bash \-c" ${ctr}
+  echo $output
+  [ "$status" -eq 0 ]
+  run buildah --debug=false inspect --type=container --format '{{.Docker.Config.Shell}}' ${ctr}
```

Comment:
Please switch this from `'{{.Docker.Config.Shell}}'` to `'{{printf "%q" .Docker.Config.Cmd }}'`, so that we can be sure that strings that should be split on whitespace are getting split correctly.

Reply:
Fixed